karen

```diff
@@ -63,6 +63,14 @@ fswatch -0 --event=PlatformSpecific $signal_dir | while read -d "" event; do
     args="${signal#$app_prefix}"
   fi
 
+  pw_prefix="passwd-"
+  if [[ "$signal" == "$pw_prefix"* ]]; then
+    trigger="$trigger_dir/passwd"
+    args="${signal#$pw_prefix}"
+    # Delete signal to remove pw from file system
+    rm "$signal_dir/$signal"
```

Comment:
If you're worried about it being stored on disk then I recommend using `/dev/shm/` for this, which is stored in RAM and will be definitely wiped at worst after reboot. If you need some file in directory, then symlink will solve this.

Reply:
I think the current solution is okay.

Reply:
Yes, not a big deal at this stage of development.